dockertest/subtest.py

```diff
@@ -291,6 +296,22 @@ def make_subsubtest_config(self, all_configs, parent_config,
         self.config = self.make_config(all_configs, parent_config,
                                        self.config_section)
 
+    def require_docker_version(self, wanted):
```

Comment:
These classes are quite crowded, what do you think about putting this in as a method on the ``Dockerversion`` class itself?  There's no requirement to use ``DockerCmd`` for such a simple call, a ``utils.run()`` or a ``subprocess.check_call()`` would be perfectly fine as well - thus eliminating the dependencies on a ``subtest`` instance as well.  Thoughts?